test/Libraries/Microsoft.Extensions.AI.Abstractions.Tests/ChatCompletion/ChatOptionsTests.cs

```diff
@@ -29,6 +32,9 @@ public void Constructor_Parameterless_PropsDefaulted()
         Assert.Null(options.AdditionalProperties);
 
         ChatOptions clone = options.Clone();
+#pragma warning disable CS0618 // Type or member is obsolete
+        Assert.Null(options.ChatThreadId);
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
TBH there's no need to add unit tests for these very temporary back-compat properties. It's not harmful to have them, but don't feel you have to do it in the future :)